Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -4275,27 +4275,23 @@ void RenderBox::computePositionedLogicalWidthUsing(SizeType widthType, Length lo
 
         const LayoutUnit availableSpace = containerLogicalWidth - (marginLogicalLeftValue + marginLogicalRightValue + bordersPlusPadding);
 
+        auto shrinkToFitLogicalWidth = [this](LayoutUnit availableLogicalWidth, LayoutUnit bordersPlusPadding) -> LayoutUnit {
```

Comment:
this probably should read
`auto shrinkToFitLogicalWidth = [&](auto availableLogicalWidth, auto bordersPlusPadding) {`

Reply:
Sure - can do it.

Reply:
Done in https://github.com/WebKit/WebKit/commit/e305b2600d3b504c55223c37217ca2b66d6317af.